journalpump/journalpump.py

```diff
@@ -551,6 +552,18 @@ def _validate_and_build_secret_filters(self, config):
 
         return secret_filters
 
+    def _configure_threshold_for_metric_emit(self, config):
+        threshold_for_metric_emit = config.get("threshold_for_metric_emit")
+        if threshold_for_metric_emit is not None:
+            try:
+                threshold = int(threshold_for_metric_emit)
+            except ValueError:
+                raise ValueError("Invalid value for threshold_for_metric_emit. Must be an integer.")
```

Comment:
Is this error message much clearer that what is given by python by default? Since we are reraising the same exception. The standard one is `ValueError: invalid literal for int() with base 10: '<input_value>'` which in some ways is clearer since it echoes the input value. It also accepts things like `int('25')` so it is not technically correct that it needs to be an integer.